python/py-nbxmpp/Portfile

```diff
@@ -26,9 +26,9 @@ master_sites        pypi:n/nbxmpp/
 
 distname            nbxmpp-${version}
 
-checksums           rmd160  b6c7088b1bf54cd272f2dea30cd42c12c7a33eff \
-                    sha256  cd73417777e4847fdd8d0d96c7cafc606952edbd2b9d52a2a72bb2aaa04d08ef \
-                    size    1516456
+checksums           rmd160  979526910665550bc7e2285c4833a466182b07fd \
+                    sha256  0fc486894e546a56d8b2baa587787aa3cc9aa3cb76912c25598c7fd11db848bd \
+                    size    141999
 
 if {${name} ne ${subport}} {
     livecheck.type  none
```

Comment:
you should specify here the appropriate build and library dependencies - look at upstream's setup.py and setup.cfg files to figure out which ones to use.